File: objectModel/Python/cdm/persistence/cdmfolder/types/projections/operation_add_type_attribute.py
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See License.txt in the project root for license information.

from cdm.persistence.cdmfolder.types.type_attribute import TypeAttribute
from cdm.persistence.cdmfolder.types.projections.operation_base import OperationBase


class OperationAddTypeAttribute(OperationBase):
    """OperationAddTypeAttribute class"""

    def __init__(self):
        super().__init__()

        self.typeAttribute = None  # type: TypeAttribute

File: objectModel/TypeScript/Persistence/CdmFolder/types/Projections/OperationAddTypeAttribute.ts
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

import { TypeAttribute } from '../TypeAttribute';
import { OperationBase } from './OperationBase';

/**
 * OperationAddTypeAttribute class
 */
export abstract class OperationAddTypeAttribute extends OperationBase {
    public typeAttribute: TypeAttribute;
}
